docs/HOWTO.md

```diff
@@ -104,7 +104,7 @@ The first thing to remember is there are **two** interesting dashboards:
 ### Create an ASK Custom Skill in your developer account
 #### Overview
  * Like most useful skills it should be a [Custom Skill](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/overviews/understanding-custom-skills)
- * Follow one of the guides ideally e.g. [Deploying a Sample Custom Skill To AWS Lambda](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/deploying-a-sample-skill-to-aws-lambda#creating-the-lambda-function-for-the-sample).
+ * Follow one of the guides ideally e.g. [Deploying a Sample Custom Skill To AWS Lambda](https://developer.amazon.com/fr/docs/custom-skills/deploy-a-sample-skill-to-aws-lambda.html).
```

Comment:
That's the (untranslated) French version. I think this is the original:

https://developer.amazon.com/docs/custom-skills/deploy-a-sample-skill-to-aws-lambda.html